primitives/keystore/Cargo.toml

```
@@ -0,0 +1,41 @@
+[package]
+name = "sp-keystore"
+version = "2.0.0"
```

```suggestion
version = "0.8.0"
```

Just because we released a bunch as 2.0, doesn't mean we should do that for new crates all the time. In particular considering that we have at least one thing in here we _do not_ want to have long-term and we just released an entirely new interface, I vote for dropping the version into the `0.x` world.